### DISCUSSION TITLE
Question about PS5's DualSense and DsHidMini + BthPS3

### DISCUSSION THREAD

**Discussion by Sly-TheFox** [nefarius/DsHidMini#292] [Support]
Hello, been using DsHidMini and BthPS3 for a long time now and has worked flawlessly ever since, but I got a DualSense as a gift and about to retire my OG PS3 controller (Godspeed, you served well o7)

And I just wanted to ask would it be recommended to uninstall both DsHidMini and BthPS3 (Or just BthPS3)?, to avoid issues or interference. Or can they just stay installed?

**Comment by nefarius** [comment]
Whatever you like; DsHidMini isn't loaded if no compatible device is present and BthPS3 will just sit there and do nothing, you can just alter the filter driver settings in the config tool and disable the filter, then your system runs like BthPS3 isn't even there:

![image](https://github.com/ViGEm/DsHidMini/assets/286631/116a99e0-be45-437c-9561-da4d848b164f)

**Reply by Sly-TheFox** [reply]
Ooh I see, thank you so very much for a quick and helpful reply, disabled PSM patching and will keep DsHidMini on as a backup.

Thank you again and I am very happy to have used your software, it always worked without issues and was very easy to use.
I hope you have a nice rest of the day! 😄

**Reply by nefarius** [reply]
Thank you kindly, and the DualSense is a nice device indeed, game on 😎

**Reply by Sly-TheFox** [reply]
No problem ❤️
First time having a newer controller since PS3 one and I have to say, I'm loving it lots, it's nice and heavy and feels really good to hold/use it, will definitely game with it lots.

**Reply by nefarius** [reply]
Make sure to check it for firmware updates though, the one's they shipped with are prone to faster battery draining and other issues.

**Reply by Sly-TheFox** [reply]
The second I plugged it in, Steam screamed at me to update the firmware and gave me a link for it. Updated without issues and everything works fine.
Thank you for the information tho! Will keep an eye out for more updates as well.
